src/lib/one-graph/cli-netlify-graph.js

```diff
@@ -217,8 +217,7 @@ const getNetlifyGraphConfig = async ({ command, options, settings }) => {
   const moduleType =
     (userSpecifiedConfig.moduleType && userSpecifiedConfig.moduleType.split(path.sep)) ||
     defaultFrameworkConfig.moduleType
-  const language =
-    (userSpecifiedConfig.language && userSpecifiedConfig.language.split(path.sep)) || autodetectedLanguage
+  const language = (userSpecifiedConfig.language && userSpecifiedConfig.language) || autodetectedLanguage
```

Comment:
```suggestion
  const language = userSpecifiedConfig.language || autodetectedLanguage
```

It seems `userSpecifiedConfig.language` is duplicated?